Libs/MRML/Widgets/qMRMLClipNodeWidget.cxx

```diff
@@ -229,6 +232,24 @@ int qMRMLClipNodeWidget::greenSliceClipState()const
 }
 
 //------------------------------------------------------------------------------
+void qMRMLClipNodeWidget::setSmoothClippingState(bool state)
+{
+  Q_D(qMRMLClipNodeWidget);
+  if (!d->MRMLClipNode.GetPointer())
+  {
+    return;
+  }
+  d->MRMLClipNode->SetUseSmoothClipping(state);
+}
+
+//------------------------------------------------------------------------------
+bool qMRMLClipNodeWidget::smoothClipState()const
```

Comment:
This will be renamed anyway, so just as a general note: keep names consistent!
For example:
setSmoothClippingState / smoothClipState => "ing" use inconsistent
setSmoothClippingState / UseSmoothClipping => "use" and "state" uses are inconsistent